assessment.md

```diff
@@ -82,39 +82,75 @@ TODO: add description for each assessment level for this skill.
 
 * Using the GitHub Flow model
 
-TODO: add description for each assessment level for this skill.
+  * :flushed: = Never has used the GitHub flow model before.
+  * :worried: = Uses it occasionally, but at inappropriate times.
+  * :neutral_face: = Student uses it often, sometimes appropriately, but sometimes inappropriately as well.
+  * :blush: = Clearly knows what the GitHub flow model is, as the student successfully uses it occasionally, but there is still room for improvement.
+  * :laughing: = Student clearly knows how and when to create branches in a project, and has an understanding of how the GitHub Flow model works.
 
 * Using the GitHub issue tracker
 
-TODO: add description for each assessment level for this skill.
+  * :flushed: = Never uses the GitHub issue tracker.
+  * :worried: = Sometimes uses the GitHub issue tracker, lacks understanding
+  * :neutral_face: = Knows how to use the issue tracker, and understands what it means, but rarely responds to issues that the team posts.
+  * :blush: = Often uses the issue tracker, but doesn't communicate enough with the team, sometimes causing confusion among the group.
+  * :laughing: = Always uses the GitHub issue tracker, and always appropriately, informing the team about their new posts to the issue tracker.
 
 * Assigning team members to issues in the GitHub issue tracker
 
-TODO: add description for each assessment level for this skill.
+  * :flushed: = Student clearly does not know how to assign members to issues.
+  * :worried: = Student knows how to add members to issues, but never does this appropriately.
+  * :neutral_face: = Student communicates with members sometimes, but needs to check with the team more often when assigning members to issues.
+  * :blush: = Student knows how and when to add members to issues, but sometimes does not communicate with the team on what they are doing.
+  * :laughing: = Communication and knowledge is superior.  Student's teammates are always aware and have agreed upon which issues they are going to be assigned to.
 
 * Creating and assigning labels to issues in the GitHub issue tracker
 
-TODO: add description for each assessment level for this skill.
+  * :flushed: =
+  * :worried: =
+  * :neutral_face: =
+  * :blush: =
+  * :laughing: =
 
 * Creating and using a repository branch
 
-TODO: add description for each assessment level for this skill.
+  * :flushed: = Never uses repository branches.
+  * :worried: = Uses repository branches when they are not appropriate.
+  * :neutral_face: = Occasionally uses branches appropriately, but needs to use them more frequently.
+  * :blush: = Demonstrates knowledge relating to the use of branches, but isn't perfect with their use.
+  * :laughing: = Always uses branches when appropriate, and uses them frequently.
```

Comment:
Again, I think it would be great to elaborate on what is the appropriate and inappropriate use of branches. Things like keeping a branch up to date with `main`, deleting a branch after a PR is merged if it's not needed, appropriately names the branches, committing to the right branch.. etc. These are some suggestions that you could incorporate in the assessment criteria.